pebblo/app/service/loader/loader_doc_service.py

```diff
@@ -290,6 +293,23 @@ def process_request(self, data):
             )
             self.db.update_data(loader_obj, app_loader_details)
 
+            # Creating loader response
+            loader_details = self.data.get("loader_details", {})
+            docs = self.data.get("docs", [])
+            loader_response_output = []
+            for doc in docs:
+                doc_obj = LoaderDocs(
+                    pb_id=doc["pb_id"],
+                    pb_checksum=hashlib.md5(doc["doc"].encode()).hexdigest(),
+                    source_path=doc["source_path_size"],
```

Comment:
Is this correct source_path=doc["source_path_size"] , Names are different that's why ..

Reply:
Can we change name to avoid confusion

Reply:
Fixed.